### PR TITLE
Change back to English tags.

### PR DESCRIPTION
**Pull request by eddieantonio**
Upon a conversation with @aarppe, even though #209 says to use `Linguistic Short` he realized that `English` is better.

(I also ran `isort` on some files...)

### CODE REVIEW

File: CreeDictionary/API/models.py [hunk 1]
```diff
@@ -417,7 +412,9 @@ def search(user_query: str) -> SortedSet[SearchResult]:
         cree_results, english_results = Wordform.fetch_lemma_by_user_query(user_query)
 
         results: SortedSet[SearchResult] = SortedSet(
-            key=cmp_to_key(partial(sort_search_result, user_query=user_query))  # type: ignore # mypy stupid
+            key=cmp_to_key(
+                partial(sort_search_result, user_query=user_query)
+            )  # type: ignore # mypy stupid
```
Comment (Madoshakalaka) [inline CreeDictionary/API/models.py]: hahahaha formatting actually broke unit test lol lol lol

**Comment by eddieantonio** [comment]
🤯🤯🤯

I'll get on it

**Comment by eddieantonio** [comment]
@Madoshakalaka, it passes type-checking and tests now 😅

I also moved the `Python-Levenshtien` dependency to `[depenencies]` as it was a dev dependency before (???)

**Comment by eddieantonio** [comment]
DERP! Just realized that `Python-Levenshtein` _is_ a dev dependency, and I forgot to install dev dependencies 😩 

This PR has been a trip!

File: CreeDictionary/API/models.py [hunk 2]
```diff
@@ -505,6 +507,22 @@ def search(user_query: str) -> SortedSet[SearchResult]:
...
 
 
+def sort_by_user_query(user_query: str) -> Callable[[Any], Any]:
+    """
+    Returns a key function that sorts search results ranked by their distance
+    to the user query.
+    """
+    # mypy doesn't really know how to handle partial(), so we tell it the
+    # correct type with cast()
+    # See: https://github.com/python/mypy/issues/1484
```
Comment (Madoshakalaka) [inline CreeDictionary/API/models.py]: I KNEW this must be a issue for it lol

File: CreeDictionary/API/models.py [hunk 3]
```diff
@@ -505,6 +507,22 @@ def search(user_query: str) -> SortedSet[SearchResult]:
...
+    return cmp_to_key(
+        cast(
```
Comment (Madoshakalaka) [inline CreeDictionary/API/models.py]: wow cast is jedi

Comment (Madoshakalaka) [reply CreeDictionary/API/models.py]: cast

**Pull request merged**